src/napari_imagej/types/converters/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""
+A module whose submodules contain additional scyjava Converters
+that are useful within napari-imagej
+
+All submodules within this module are DYNAMICALLY IMPORTED; this allows
+automatic discovery of all Converters.
+
+The previous approach declared all Converters in one module, imported
```

Comment:
I would suggest not to talk about previous approaches, which most readers are unlikely to be familiar with. Instead, just focus on how things are *laid out now*.

Reply:
Removed in e3a5ce8